app/src/main/java/com/example/countryinfo/MainActivity.kt

```diff
@@ -46,4 +59,17 @@ class MainActivity : AppCompatActivity(), CountryPreviewFragment.ItemClickedList
         }
         return super.onOptionsItemSelected(item)
     }
+
+    private fun setOrientation(){
+        requestedOrientation = if(resources.getBoolean(R.bool.is_tablet)){
+            ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
+        } else{
+            ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
+        }
+    }
```

Comment:
It's not gonna help in our case since we need portrait only mode for phones and landscape only mode for tablets and "nosensor" forces portrait mode everywhere.